### DISCUSSION TITLE
fa/manager/configuration-and-advanced-settings/how-to-import-xui-to-hiddifymanager/

### DISCUSSION THREAD

**Discussion by giscus** [hiddify/Hiddify-Manager#4160] [Content Help کمک به اصلاح محتوا]
# fa/manager/configuration-and-advanced-settings/how-to-import-xui-to-hiddifymanager/

The proxy manager platform.

https://hiddify.com/fa/manager/configuration-and-advanced-settings/how-to-import-xui-to-hiddifymanager/

**Comment by zarvan9** [comment]
سلام من با این ارور مواجه میشم

failed to import xui data: Error: (MySQLdb.ProgrammingError) (1146, "Table 'hiddifypanel.inbounds' doesn't exist")
[SQL: select * from inbounds]
(Background on this error at: https://sqlalche.me/e/20/f405)

**Reply by TomCat-Hub** [reply]
درود بر شما دوست گلم
وقت بخیر

لطفا سرور رو ریبیلد کنید و یکبار دیگه و با سرور اوبونتو 22.04 تلاش کنید و اگر باز هم با مشکلی روبرو شدید توی گروه تلگرام هیدیفای و تاپیک هیدیفای منیجر پیام بفرستید تا دوستان برسی کنند.

آدرس گروه تلگرام هیدیفای:
https://t.me/hiddify_board

باتشکر از انتخاب شما
مانا باشید

**Comment by Amirhosseinamnn** [comment]
سلام این رو اجرا کردم ولی ارور میده و انجام نمیشه
ارور توکن
